cli/version.rs

```diff
@@ -1,9 +1,15 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
-pub const DENO: &str = env!("CARGO_PKG_VERSION");
 pub const GIT_COMMIT_HASH: &str = env!("GIT_COMMIT_HASH");
 pub const TYPESCRIPT: &str = crate::js::TS_VERSION;
 
+pub fn deno() -> String {
+  let semver = env!("CARGO_PKG_VERSION");
+  option_env!("DENO_CANARY").map_or(semver.to_string(), |_| {
```

Comment:
Is `DENO_CANARY` env variable defined anywhere?

Reply:
as discussed on the discord with luca, it will be added to ci